package.json

```diff
@@ -8,7 +8,9 @@
     "test": "",
     "build": "npm run clean; webpack --env=prod --colors",
     "clean": "rm -rf dist",
-    "dist:server": "pushd dist; python -m SimpleHTTPServer; popd"
+    "dist:server": "pushd dist; python -m SimpleHTTPServer; popd",
+    "dev-deploy": "aws s3 sync dist/ s3://mapstaging",
```

Comment:
I gave these different names to make it a little harder to accidentally deploy prod :grin: